go/mysqlconn/replication/binlog_event_rbr.go

```diff
@@ -303,6 +302,17 @@ func cellLength(data []byte, pos int, typ byte, metadata uint16) (int, error) {
 	}
 }
 
+// printTimestamp is a helper method to print a timestamp.
+func printTimestamp(v uint32) string {
```

Comment:
You could change this to appendTimestamp and use `fmt.Fprintf(bytes.Buffer, ...)`